go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/OpenNebula/terraform-provider-opennebula
 go 1.13
 
 require (
-	github.com/OpenNebula/one/src/oca/go/src/goca v0.0.0-20201006100728-132291f85c17
+	github.com/OpenNebula/one/src/oca/go/src/goca v0.0.0-20200610135545-ec5f33cd58ba
```

Comment:
It seems you are going back on the goca.

Reply:
I mess that while rebasing the changes I'll revert that change.